actions/ci-beholder-validator/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "ci-beholder-validator",
+  "version": "0.0.0",
```

Comment:
Please add a changeset before merging this, it will bump this version and create a git tag for you.

```
pnpm install
pnpm changeset
```

Reply:
will do.